colorization/src/pipeline.py

```diff
@@ -49,11 +49,28 @@ def colorize_image(image_path_input, image_path_output):
     The directories already exists, and the image can be directly
     written to the given output path.
     """
-
     kModelWidth = numpy.uint32(224)
     kModelHeight = numpy.uint32(224)
     KMODELPATH = os.path.join(os.path.abspath(os.path.dirname(__file__)),
                               "../../model/colorization.om")
+
+    #  check if the input path is valid
+    if not utils.IsPathExist(image_path_input):
+        print("input path does not exist.")
+        return FAILED
+    if not utils.IsDirectory(image_path_input):
+        print("input path is not a file.")
+        return FAILED
+
+    #  check if the path for model is valid
+    if not utils.IsPathExist(KMODELPATH):
+        print("model path does not exist.")
+        return FAILED
+    if not utils.IsDirectory(KMODELPATH):
+        print("model path is not a file.")
+        return FAILED
+
```

Comment:
it is currently the function name in utils.py, when... then the utils file need to be changed

Reply:
@Emma-An94 Emma, I would also suggest using standard lib instead of "'IsDirectory" in utils.py at the moment. "os.path.isfile(path)"  would work for you.